tsk/base/mymalloc.c

```diff
@@ -44,10 +44,15 @@ tsk_malloc(size_t len)
 void *
 tsk_realloc(void *ptr, size_t len)
 {
-    if ((ptr = realloc(ptr, len)) == 0) {
```

Comment:
I don't see what these changes fix?  Aren't they just the same thing, but more verbose?

Reply:
If/when realloc failed, ptr is assigned to NULL. The original ptr is lost so a memory leak.
This was reported by VLD.

Reply:
https://stackoverflow.com/questions/4548385/realloc-leaking-memory

Reply:
Oh, we should make a comment then to that effect.  Otherwise, someone will return it back to what it was.

Reply:
done